src/javaRosa.js

```diff
@@ -892,6 +892,9 @@ define([
             var itextItem = widget.getItextItem();
             if (itextItem) {
                 if (widget.isDefaultLang) {
+                    if (!value) {
+                        value = widget.getPlaceholder();
+                    }
```

Comment:
That should change. The itext form is where the writer will look when writing that value: https://github.com/dimagi/Vellum/blob/fix-no-placeholder/src/javaRosa.js#L1610-L1619

This is a bit of a corner case though, as this only happens whenever we have one language.